packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -0,0 +1,1391 @@
+---
+title: Brands
+category: management
+---
+
+# Brands API
+
+The Okta Brands API provides operations to manage branding for your organization.
```

Comment:
Proposal:

`The Okta Brands API allows customization of the look and feel of your end-user pages. Upload your ]colors, background image, logo, and favicon, and then publish these assets directly to the Okta-hosted sign in page, error pages, email templates, and end-user dashboard.